api/app/src/main/kotlin/packit/model/dto/PacketGroupDto.kt

```diff
@@ -2,5 +2,6 @@ package packit.model.dto
 
 data class PacketGroupDto(
     val name: String,
+    val latestDisplayName: String,
```

Comment:
I'm a little bit uneasy about the inconsistency between "displayName" for `PacketGroupDisplay` and "latestDisplayName" for `PacketGroupDto`. I guess there's potentially a subtle difference here which justifies the discrepancy in that `PacketGroupDisplay` is an ephemeral sort of thing so its display name is just the display name now, whereas `PacketGroup` persists and can have different "latestDisplayNames" over time..? But i think I'd still be inclined to just use the same prop for both to keep things simple.

Reply:
Renamed all to 'latest...'